index.html

```diff
@@ -0,0 +1,192 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="assets/styles/app.css">
+    <title>Document</title>
+</head>
+<body class="page">
+        <main>
+          <section>
+            <header class="header">
+              <img src="assets/icons/menu.svg" alt="Menu">
+            </header>
+           <div class="headline">
+                 <div class="background-image">
+                   <img src="assets/icons/header.svg" alt="header">
+                 </div> 
+                 <h1 class="lines"> Hey there <br>
+                   I'm THIERRY <br>
+                   <div class="middle">I'm a Software Developer</div>
+                 </h1>
+                   <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                       don’t hesitate to contact me.</p>
+                   <ul class="socials">
+                     <li><img src="assets/icons/github.svg" alt="github"></li>
+                     <li><img src="assets/icons/linkeldn.svg" alt="linkeldn"></li>
+                     <li><img src="assets/icons/twitter.svg" alt="twitter"></li>
+                     <li><img src="assets/icons/angellist.svg" alt="angellist"></li>
+                     <li><img src="assets/icons/medium.svg" alt="medium"></li>
+                   </ul>
+                   <div class="button">
+                     <img src="assets/icons/normalbottun.svg" alt="normalbottun">
+                   </div>
+             </div>
+          </section>
+                  <section>
+                    <h2 class="hdescription">MY RECENT <br> WORKS</h2>
+                  </section>
+                  <section class="section-placeholder">
+                    <div  class="placeholder-container">
+                      <!-- <article class="placeholder-box"> -->
+                        <img src="assets/icons/placeholder.svg" alt="placeholder" class="placeholder">
+                        <div class="text-container">
+                          <h1>Multi-Post Stories <br> Gain+Glory</h1>
+                          <ul class="text">
+                            <li class="text-ruby">Ruby on Rails</li>
+                            <li class="text-css">CSS</li>
+                            <li class="text-java">JavaScript</li>
+                            <li class="text-html">HTML</li>
+                          </ul>
```

Comment:
- [ ] Nice work at refactoring this 👏 , however, I suggest that you stick to the design in the template and align the horizontal, as it is essential
 for a front-end engineer to be able to create an app based on a design without any personal touches 👨‍💻 

| Your App| Figma design|
| --- | --- |
|![design](https://user-images.githubusercontent.com/102636489/228941996-7fba4120-c888-4a1f-9989-5231f2685770.png)]| ![yours](https://user-images.githubusercontent.com/102636489/228941683-e99bb617-5290-4d8c-b3c8-fe6efdf81d16.png) |